### PR TITLE
Add free() to libsession wrappers

### PR DESCRIPTION
**Pull request by yougotwill**
`free()` clears wrappers that are stored in memory and haven't been saved to the database yet, there is no direct call to `libsession` but we want the typings

### CODE REVIEW

File: user/contacts.d.ts [hunk 1]
```diff
@@ -55,6 +57,7 @@ declare module 'libsession_util_nodejs' {
 
   export type ContactsConfigActionsType =
     | ['init', Uint8Array, Uint8Array | null]
+    | MakeActionCall<UserConfigWrapper, 'free'>
```
Comment (Bilb) [inline user/contacts.d.ts]: this should be 
```suggestion
    | MakeActionCall<ContactsWrapper, 'free'>
```

File: user/convovolatile.d.ts [hunk 2]
```diff
@@ -60,6 +62,7 @@ declare module 'libsession_util_nodejs' {
 
   export type ConvoInfoVolatileConfigActionsType =
     | ['init', Uint8Array, Uint8Array | null]
+    | MakeActionCall<UserConfigWrapper, 'free'>
```
Comment (Bilb) [inline user/convovolatile.d.ts]:
```suggestion
    | MakeActionCall<ConvoInfoVolatileWrapper, 'free'>
```

File: user/contacts.d.ts [hunk 3]
```diff
@@ -6,6 +6,8 @@
 declare module 'libsession_util_nodejs' {
   type ContactsWrapper = BaseConfigWrapper & {
     init: (secretKey: Uint8Array, dump: Uint8Array | null) => void;
+    /** This function is used to free wrappers from memory only */
+    free: () => void;
     get: (pubkeyHex: string) => ContactInfo | null;
     set: (contact: ContactInfoSet) => void;
     getAll: () => Array<ContactInfo>;
```
Comment (Bilb) [inline user/contacts.d.ts:13]: you didn't commit any cpp files update. Surely that's wrong.

**Review by Bilb — CHANGES_REQUESTED** [review]

File: user/contacts.d.ts (hunk 3 above)
Comment (yougotwill) [reply user/contacts.d.ts:13]: as mentioned above free() is only used in the frontend for wrappers that are stored in memory and haven't been saved to the database yet, there is no direct call to libsession but we want the typings

File: user/contacts.d.ts (hunk 1 above)
Comment (yougotwill) [reply user/contacts.d.ts]: fixed

File: user/convovolatile.d.ts (hunk 2 above)
Comment (yougotwill) [reply user/convovolatile.d.ts]: fixed

**Comment by Bilb** [comment]
LGTM

**Pull request merged**